[2025-02-14T04:16:13.700+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T04:16:13.740+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: web_to_snowflake.scrape_weather_data manual__2025-02-14T04:15:57.055594+00:00 [queued]>
[2025-02-14T04:16:13.750+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: web_to_snowflake.scrape_weather_data manual__2025-02-14T04:15:57.055594+00:00 [queued]>
[2025-02-14T04:16:13.750+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-14T04:16:13.767+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): scrape_weather_data> on 2025-02-14 04:15:57.055594+00:00
[2025-02-14T04:16:13.782+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=349) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T04:16:13.783+0000] {standard_task_runner.py:72} INFO - Started process 350 to run task
[2025-02-14T04:16:13.784+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'web_to_snowflake', 'scrape_weather_data', 'manual__2025-02-14T04:15:57.055594+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/web_to_snowflake.py', '--cfg-path', '/tmp/tmpi4y9x0h8']
[2025-02-14T04:16:13.788+0000] {standard_task_runner.py:105} INFO - Job 75: Subtask scrape_weather_data
[2025-02-14T04:16:13.844+0000] {task_command.py:467} INFO - Running <TaskInstance: web_to_snowflake.scrape_weather_data manual__2025-02-14T04:15:57.055594+00:00 [running]> on host 72b71de099bb
[2025-02-14T04:16:13.928+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='web_to_snowflake' AIRFLOW_CTX_TASK_ID='scrape_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T04:15:57.055594+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T04:15:57.055594+00:00'
[2025-02-14T04:16:13.932+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-14T04:16:13.959+0000] {logging_mixin.py:190} INFO - Scraped data: {'temperature': 72, 'humidity': 65, 'date': '2025-02-14', 'temperature_celsius': 22.22}
[2025-02-14T04:16:13.959+0000] {python.py:240} INFO - Done. Returned value was: Data scraped successfully
[2025-02-14T04:16:13.979+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-14T04:16:13.980+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=web_to_snowflake, task_id=scrape_weather_data, run_id=manual__2025-02-14T04:15:57.055594+00:00, execution_date=20250214T041557, start_date=20250214T041613, end_date=20250214T041613
[2025-02-14T04:16:14.011+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T04:16:14.037+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-14T04:16:14.038+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
